src/bot.py

```diff
@@ -29,6 +29,10 @@ async def ping(ctx):
 async def bing(ctx):
     await ctx.send("bong")
 
+@bot.command(name="ding")
+async def dong(ctx):
+    await ctx.send("8====D")
```

Comment:
change this

Reply:
I disagree

Reply:
For consistency, the string being sent should "dong"

Reply:
I agree with @gobisa